articles/TOC.yml

```
@@ -1,23 +1,21 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Overview
+- name: Get started with quantum computing
   items:
-  - name: Get started
-    href: welcome.md
-  - name: Five questions about quantum computing
-    items:
-    - name: What is quantum computing?
-      href: overview/what-is-quantum-computing.md
-    - name: What can a quantum computer do?
-      href: overview/quantum-computers.md
-    - name: Why learn quantum computing?
-      href: overview/why-learn-quantum-computing.md
-    - name: What is Q#?
-      href: overview/what-is-qsharp.md
-    - name: How to learn quantum computing?
-      href: overview/how-to-learn-quantum-computing.md
-- name: Install and update
+  - name: What is quantum computing?
+    href: overview/what-is-quantum-computing.md
+  - name: What can a quantum computer do?
+    href: overview/quantum-computers.md
+  - name: Why learn quantum computing?
+    href: overview/why-learn-quantum-computing.md
+  - name: What are Q# and QDK?
+    href: overview/what-is-qsharp.md
+  - name: How can I learn more?
+    href: overview/how-to-learn-quantum-computing.md
+- name: Get the QDK
```

Something to check in preview rendering, but if you have enough room to spell out "Quantum Development Kit" here I would suggest that could be helpful.
```suggestion
- name: Get the Quantum Development Kit
```

---

Agreed, if it fits in the preview it's ideal. Giving it a shot now